playbooks/vars/defaults.yaml

```diff
@@ -95,10 +95,10 @@ tripleo_repos_repos:
   - current-tripleo-dev
   - ceph
 
-ceph_enabled: false
+ceph_enabled: true
```

Comment:
this is too opinionated, IMO.

- in real life, people won't use loop devices to test OCP clusters in VMs backed by Ceph loop devices, the cluster won't start it's too slow
- not everyone has 100GB of available

This feature is useful to test things like "hey can I create a volume" or testing Manila/Ceph integration.
But IMO we shouldn't enable this by default.

Reply:
I strongly disagree about this 😉

Most devs will want ceph and won't want to reinstall because they forgot to enable it, and anybody using a DSAL box has way more than 100G disk going spare.

I think disabling ceph is an interesting, but niche configuration. I'm happy for the niche configuration to require additional configuration. I don't want everybody else to require it.

Reply:
**All** DSAL users have > 1 disk and will dedicate a disk (e.g. */dev/sdb*) to create the LVM configuration and have good performances using direct block storage, vs using a loop device which we all know doesn't provide great I/O results.

Reply:
If you merges https://github.com/shiftstack/dev-install/pull/52, i'm fine with this change.